core/storage/statistics/gae_models.py

```diff
@@ -613,7 +793,8 @@ def _insert_submitted_answers_unsafe(
             last_shard.submitted_answer_list,
             last_shard.accumulated_answer_json_size_bytes,
             new_submitted_answer_dict_list)
-        new_shard_count = main_shard.shard_count + len(sharded_answer_lists) - 1
+        new_shard_count = main_shard.shard_count + \
```

Comment:
Don't end sentences using backslashes. Read the coding style guide for the best practices - [Coding Style Guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#python).

Reply:
done

Reply:
This for line continuation.

Reply:
@abhishekarya286 If you need to do line continuation, open a parentheses and continue in the next line.

Reply:
Okay, I edited this.